### ISSUE TITLE
Where is changelog for 3.19.0

### ISSUE THREAD

**Issue by stoivo** [testdouble/testdouble.js#518]
Changelog.md most recent is 3.18.0

**Comment by giltayar** [comment]
Good question. Since I've been doing the latest releases, then I must have skipped this. 

@searls IIUC the change to `changelog.md` is manual? If so, I'll update it with the relevant information.

**Comment by searls** [comment]
Yep, changelog is manual nowOn Oct 20, 2023, at 02:39, Gil Tayar ***@***.***> wrote:﻿
Good question. Since I've been doing the latest releases, then I must have skipped this.
@searls IIUC the change to changelog.md is manual? If so, I'll update it with the relevant information.
